### DISCUSSION TITLE
Toggle Javascripts with Jarvis

### DISCUSSION THREAD

**Discussion by PierrePetite** [Zefau/ioBroker.jarvis#735] [Ideas / Feature Requests]
As I am using Blocky-JS to build Automations, I would love to be able to Active and Deactive the Scripts in Jarvis. With other VIS I am able to toggle them between on and off. But with Jarvis I am not able to select them as Device.

**Comment by Zefau** [comment]
This will be added with https://github.com/Zefau/ioBroker.jarvis/issues/38
